### ISSUE TITLE
promise/param-names throwing unnecessary errors?

### ISSUE THREAD

**Issue by yourfavorite** [xjamundx/eslint-plugin-promise#54]
I could just disable the rule but I'd like to understand why this error is being thrown. Thanks!

Below is my code (the entire file). The error underlines `const`.
```
const settings = {};
module.exports = { settings };
```

Here is my .eslintrc
```
{
  "env": {
    "node": true,
    "browser": true,
    "es6": true
  },
  "extends": "eslint:recommended",
  "parserOptions": {
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true,
      "jsx": true
    }
  },
  "plugins": [
    "react",
    "node"
  ],
  "rules": {
    "indent": ["error", 2, { "SwitchCase": 1 }],
    "linebreak-style": ["error", "unix" ],
    "quotes": ["error", "single"],
    "semi": ["error", "always"],
    "arrow-parens": ["off"],
    "consistent-return": "off",
    "comma-dangle": "off",
    "no-console": "off",
    "no-use-before-define": "off",
    # "no-underscore-dangle": "off",
    "generator-star-spacing": "off",
    "promise/param-names": 2,
    "promise/always-return": 2,
    "promise/catch-or-return": 2,
    "promise/no-native": 0,
    "react/jsx-no-bind": "off",
    "react/jsx-filename-extension": ["error", { "extensions": [".js", ".jsx"] }],
    "react/prefer-stateless-function": "off",
    "react/forbid-prop-types": "off",
    "flowtype-errors/show-errors": 2
  }
}
```

**Comment by macklinu** [comment]
@yourfavorite did you get this resolved? It looks like the "plugins" section of your `.eslintrc` file is missing the "promise" plugin, which may be causing the error you are experiencing. Check out the [Usage](https://github.com/xjamundx/eslint-plugin-promise#usage) section in the docs for the recommended usage syntax.

**Comment by macklinu** [comment]
Closing this due to inactivity - please reopen if you are experiencing this issue still, or open another issue if you find another bug. Thank you.
